libs/ledgerjs/packages/cryptoassets/src/abandonseed.ts

```diff
@@ -1,13 +1,16 @@
 import invariant from "invariant";
+import { CryptoCurrencyId } from "@ledgerhq/types-cryptoassets";
 
 /**
  * these are either "dead"/"burn" addresses OR "abandon" seed addresses.
  * These addresses are PUBLIC addresses
  * We use them for tests and also for dry-run estimations
  * DO NOT USE AS RECIPIENT OR SIGN TRANSACTIONS INTO THEM
  */
-const abandonSeedAddresses: Record<string, string> = {
+const abandonSeedAddresses: Partial<Record<CryptoCurrencyId, string>> = {
```

Comment:
Does this work, i am not sure ? 
{[k: CryptoCurrencyId] : string }
I feel like this is more readable than Partial<Record<CryptoCurrencyId, string>>

Reply:
Yes the syntax is correct and works as intentend
`{ [k in CryptoCurrencyId]: string }` is also valid however it requires **all** cryptocurrencyIds where here we only want a part of it thus the partial.
`Partial<{ [k in CryptoCurrencyId]: string }>` works fine but I prefer the `Record` syntax here